2022/typescript/day07.ts

```
@@ -1,61 +1,61 @@
-      continue;
+    } else if (commandOrDirEnt === 'dir') {
+      if (structure[`${path}.${arg}`] === undefined) structure[`${path}.${arg}`] = 0
+      continue
+    } else if (commandOrDirEnt === 'ls') {
+      continue
     }
 
-    const fileSize = Number(commandOrDirEnt);
-    if (!isNaN(fileSize)) structure[path] += fileSize;
+    const fileSize = Number(commandOrDirEnt)
+    if (!isNaN(fileSize)) structure[path] += fileSize
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)